check.go

```diff
@@ -199,7 +203,27 @@ func (check *Check) web(name *string, sleep *time.Duration) {
 
 // The actual HTTP GET.
 func (check *Check) fetch() error {
-	resp, err := http.Get(check.Web)
+	var resp *http.Response
+	var err error
+	method := "GET"
+	if len(check.Headers) > 0 {
+		client := http.Client{}
+		var body io.Reader
+		if len(check.Body) > 0 {
```

Comment:
Correct, this is an error.  It should be checking for the length of `check.Body`.

We can back out the change for Body; I don't use it myself, and included it only to cover some use cases that might come up.